### PR TITLE
feat: add a new variant to custom accordion

### PR DESCRIPTION
**Pull request by levi-zustiak**
# Description

### Issue
Needed an additional variant in the `Accordion` component

[Link to Linear Issue](https://linear.app/jesus-film-project/issue/ENG-622/add-a-new-variant-to-custom-drawer-accordion)

### Solution
Add a new variant to `Accordion` component

# External Changes

# Additional information

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/RadioOption/RadioOption.tsx [hunk 1]
```diff
@@ -33,7 +33,6 @@ export function RadioOption({
         icon={<LinkIcon />}
         name={t('Action')}
         value={selectedAction.label}
-        testId="RadioOption"
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/RadioOption/RadioOption.tsx:36]: Do we need this removed?

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -11,24 +11,26 @@ import { ReactElement, ReactNode } from 'react'
...
-  testId?: string
+}
+
+type AccordionProps = {
+  id: string
+  icon: ReactElement
   param?: string
   children: ReactNode
-}
+} & (StaticLabelProps | DynamicLabelProps)
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx:30]: Having this kind of variants is a little over engineered for this change I think lol. Do you think we can just make `value` an optional prop, and show the visual differences based off its shown or not?

Can we also add a unit test for this too?

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/RadioOption/RadioOption.tsx (hunk 1 above)
Comment (levi-zustiak) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/RadioOption/RadioOption.tsx:36]: The `testId` prop wasn't used in the component. A `testId` is generated using the `id` prop

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx (hunk 2 above)
Comment (levi-zustiak) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx:30]: Yeah we could use the value prop, but I like this version for the explicit definition so it's harder to misuse and a little more concise on what's expected. If you still think it's overboard I can go ahead and switch implementations 👍

Comment (csiyang) [reply apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx:30]: @levi-zustiak Yeah lets switch to using the value prop

File: apps/api-videos/src/app/modules/importer/importerVideoVariants/importerVideoVariants.service.spec.ts [hunk 3]
```diff
@@ -67,15 +67,17 @@ describe('ImporterVideoVariantsService', () => {
           id: 'mockId',
           languageId: '529',
           slug: 'Variant-Title/english',
-          videoId: 'videoId'
+          videoId: 'videoId',
```
Comment (csiyang) [inline apps/api-videos/src/app/modules/importer/importerVideoVariants/importerVideoVariants.service.spec.ts:70]: what are these changes??

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx [hunk 4]
```diff
@@ -75,13 +74,19 @@ export function Accordion({
       >
         <Stack spacing={3} alignItems="center" direction="row">
           {icon}
-          <Box sx={{ maxWidth: 92, overflow: 'hidden' }}>
-            {name != null && (
-              <Typography variant="caption" color="text.secondary" noWrap>
+          <Box sx={{ maxWidth: '24ch', overflow: 'hidden' }}>
+            {value === undefined ? (
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/Accordion/Accordion.tsx:78]:
```suggestion
            {value != undefined ? (
```

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/api-videos/src/app/modules/importer/importerVideoVariants/importerVideoVariants.service.spec.ts (hunk 3 above)
Comment (levi-zustiak) [reply apps/api-videos/src/app/modules/importer/importerVideoVariants/importerVideoVariants.service.spec.ts:70]: Broken test from a PR that was merge just a little while ago. Just fixed it so the ci/cd passes

**Review by csiyang — APPROVED** [review]

**Pull request merged**